docs/backend-api/resources/tracking/tracker/rules/rule_types.md

```diff
@@ -1398,12 +1421,38 @@ A rule that triggers when value of a chosen measurement sensor gets into or out
 
 #### extended parameters
 
-| name      | description               | type |
-|-----------|---------------------------|------|
-| threshold | A threshold for a sensor. | int  |
-| sensor_id | ID of a tracked sensor.   | int  |
-| min       | A minimum range value.    | int  |
-| max       | A maximum range value.    | int  |
+| name            | description                                                                                                      | type        |
+|-----------------|------------------------------------------------------------------------------------------------------------------|-------------|
+| sensor_id       | ID of a tracked sensor. Only specified if `tracker_params` is not specified.                                     | int         |
+| threshold       | A threshold for a sensor. Ignored if `tracker_params` is specified.                                              | int         |
+| min             | A minimum range value. Ignored if `tracker_params` is specified.                                                 | int         |
+| max             | A maximum range value. Ignored if `tracker_params` is specified.                                                 | int         |
+| tracker_params  | An optional object. Specifies a list of parameters to be tracked in the rule, including for different trackers.  | JSON object |
+
+#### tracker_params
+
+| name            | description               | type        |
+|-----------------|---------------------------|-------------|
+| sensor_id       | ID of a tracked sensor.   | int         |
+| threshold       | A threshold for a sensor. | int         |
```

Comment:
Could you add a note that if the parameter is omitted or null, then the default value of 0.03 is used? And in the table above.